Description: Avoid changing log directory ownership on each service startup 
 Removes obsolete ExecStartPre entries from mosquitto.service
Author: Joachim Zobel <jz-2027@heute-morgen.de>
Bug: https://github.com/eclipse-mosquitto/mosquitto/issues/3344
Last-Update: 2025-08-17 
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/service/systemd/mosquitto.service.notify
+++ b/service/systemd/mosquitto.service.notify
@@ -10,10 +10,6 @@
 ExecStart=/usr/sbin/mosquitto -c /etc/mosquitto/mosquitto.conf
 ExecReload=/bin/kill -HUP $MAINPID
 Restart=on-failure
-ExecStartPre=/bin/mkdir -m 740 -p /var/log/mosquitto
-ExecStartPre=/bin/chown mosquitto:mosquitto /var/log/mosquitto
-ExecStartPre=/bin/mkdir -m 740 -p /run/mosquitto
-ExecStartPre=/bin/chown mosquitto:mosquitto /run/mosquitto
 
 [Install]
 WantedBy=multi-user.target
